client/src/sections/landingPage/benefits.tsx

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import Image from 'next/image';
+import type { FC } from 'react';
+
```

Comment:
- use Kebab case for directory name
- then use PascalCase for component name

![image](https://user-images.githubusercontent.com/73863685/227911777-ab29ea84-6b8a-442f-b9f8-73ded536c223.png)


```
client/src/sections/landing-page/Benefits.tsx
```

Apply to all Related